<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatlabCameratestClass</class>
 <widget class="QMainWindow" name="MatlabCameratestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatlabCameratest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Cam1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget2" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget4" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget5" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>430</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget6" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget3" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>10</y>
      <width>65</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>connected?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>waiting...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect_Controller1">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>10</y>
      <width>116</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect_Controller</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="position_x">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="position_y">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="position_z">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="zposition_x">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="zposition_y">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="zposition_z">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="position_u">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="position_v">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="position_w">
     <property name="text">
      <string>0.0000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="zposition_u">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="zposition_v">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="zposition_w">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="Reference">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reference</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MoveToTarget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Move To Target</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_X">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Z">
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>20</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_U">
     <property name="text">
      <string>U</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_V">
     <property name="text">
      <string>V</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_W">
     <property name="text">
      <string>W</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Cam2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Track1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Track1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Picture1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Picture2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>190</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>190</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Open">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>74</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_8">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>81</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="cbx_cam1Rpoint">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Cam1RPoint</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_cam1Pnp">
     <property name="text">
      <string>Cam1PnP</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_cam1Ukf">
     <property name="text">
      <string>Cam1Ukf</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_cam2Rpoint">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Cam2RPoint</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_cam2Pnp">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cam2PNP</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbx_cam2Ukf">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cam2Ukf</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MatlabCameratest.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_Open</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>btn_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Connect_Controller1</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>ConnectSixAxis()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Reference</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>ReferenceSixAxis()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MoveToTarget</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>MoveToTarget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>870</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Track1</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Track1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Picture1</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>picture1Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Picture2</sender>
   <signal>clicked()</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>picture2Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam1Rpoint</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam1RPoint(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>917</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam1Pnp</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam1Pnp(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam1Ukf</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam1Ukf(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>915</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>892</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam2Rpoint</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam2RPoint(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>944</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>962</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam2Pnp</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam2Pnp(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>928</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbx_cam2Ukf</sender>
   <signal>clicked(bool)</signal>
   <receiver>MatlabCameratestClass</receiver>
   <slot>Cam2Ukf(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>946</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_open()</slot>
  <slot>ConnectSixAxis()</slot>
  <slot>ReferenceSixAxis()</slot>
  <slot>MoveToTarget()</slot>
  <slot>Exit()</slot>
  <slot>Track1()</slot>
  <slot>picture1Clicked()</slot>
  <slot>picture2Clicked()</slot>
  <slot>Cam1RPoint(bool)</slot>
  <slot>slot2()</slot>
  <slot>Cam1Pnp(bool)</slot>
  <slot>slot3()</slot>
  <slot>Cam1Ukf(bool)</slot>
  <slot>Cam2RPoint(bool)</slot>
  <slot>Cam2Pnp(bool)</slot>
  <slot>Cam2Ukf(bool)</slot>
 </slots>
</ui>
